config/debian/dpkg_client/debian/rules

```diff
@@ -43,10 +28,11 @@ override_dh_strip:
 	# Necessary otherwise the PyInstaller build executables will not run
 
 override_dh_makeshlibs:
-	# Prevent useless call to ldconfig
-	dh_makeshlibs -n
+	# These add a lot of unnecessary dependencies to the deb package because
+	# it just looks for .so files and adds them as dependencies. Since we
+	# are shipping our .so's with Pyinstaller this is unnecessary
 
 override_dh_shlibdeps:
-	# Point debhelper to the locally included shared libraries.
-	dh_shlibdeps --dpkg-shlibdeps-params=--ignore-missing-info -l %(ClientBuilder.target_dir)
-
+	# These add a lot of unnecessary dependencies to the deb package because
+	# it just looks for .so files and adds them as dependencies. Since we
```

Comment:
I suspect this will be fine if the dependencies aren't there anymore.  The problem was basically that running the deb tools stuff RHEL/Centos doesn't know how to properly resolve dependencies since it isn't a real apt/dpkg system so ignore-missing-info just tells it to not fatally die when attempting to build the .deb.  

The impact here would be that RHEL/Centos systems can't build debs properly.  I'm not even sure that RHEL7 provides the proper tools to do it with or without this change anyways.